database/migrations/2019_11_29_103335_create_dinnerforms_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateDinnerformsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('dinnerforms', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('restaurant');
+            $table->string('description');
+            $table->string('url');
+            $table->dateTime("start");
+            $table->dateTime('end');
+            $table->timestamps();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::dropIfExists('dinnerform');
```

Comment:
That's the wrong table name
```suggestion
        Schema::dropIfExists('dinnerforms');
```